kind/pkg/cluster/cluster.go

```diff
@@ -130,7 +135,10 @@ func (c *Context) Delete() error {
 
 // provisionControlPlane provisions the control plane node
 // and the cluster kubeadm config
-func (c *Context) provisionControlPlane(nodeName string, config *CreateConfig) (kubeadmConfigPath string, err error) {
+func (c *Context) provisionControlPlane(
+	nodeName string,
+	config *config.Config,
```

Comment:
Were the added line breaks in preparation for a load of new args being added to this method? 😄

Reply:
I hope not 😬, it just felt long when I was working on this. 🤔